labs/shipped-import-project/3.md

```diff
@@ -0,0 +1,18 @@
+# Shipped - Import Existing project to shipped
+
+1 Check Build logs, your should be able to check build in progress for shipped commits to your repo during "Save Project"
+
```

Comment:
We need to dicuss with @dorlok how we can avoid duplication of steps in all learning labs. 
- build, env create, deploy are common steps needed in multiple labs. 
- We need to have some way of reusing same .md file. Else if one screenshot is changed that would require change in multiple learning labs.